test/test_um2netcdf.py

```diff
@@ -1,6 +1,8 @@
 import unittest.mock as mock
 from dataclasses import dataclass
 from collections import namedtuple
+import numpy as np
+from iris.exceptions import CoordinateNotFoundError
 
 import umpost.um2netcdf as um2nc
```

Comment:
Out of interest, how am I meant to get this working for testing? I can't `pip install -e` the umpost directory in its current state, so I'm getting test collection errors.

Reply:
Whoops, due to prior history, this project has started oddly. Spencer is using a `gadi`'s environment & I'm reusing a local virtualenv from another project. There's been no setup docs until last week, when I grabbed docs from a related project. Do you feel like yet another PR review for docs? https://github.com/ACCESS-NRI/um2nc-standalone/pull/62
(these docs are missing `pip -e`)

Reply:
I do like me some good docs...